packages/volto/news/4058.bugfix

```diff
@@ -0,0 +1 @@
+Fix the size of Select Menu in SelectWidget to prevent users from having to scroll both the main modal and dropdown menu to view the field choices by adding a menuPortalTarget prop. @victorchrollo14
```

Comment:
Change logs should avoid implementation details, as the code covers that.

```suggestion
In a modal with a `SelectWidget`, the parent modal no longer requires scrolling to view the choices in the select's options list. @victorchrollo14
```

Reply:
Ok. Thank you. 
Shall I wait for the maintainers to review and make all the suggested changes in a single commit.